cmd/storage-datatypes.go

```diff
@@ -502,6 +502,10 @@ type RenameDataResp struct {
 	OldDataDir string // contains '<uuid>', it is designed to be passed as value to Delete(bucket, pathJoin(object, dataDir))
 }
 
+type CheckPartsResp struct {
+	Errs []string
+}
+
```

Comment:
I don't really like all the error -> string magic. 

Can't we just make a type safe response:

```suggestion
// CheckPartsResp is the result of CheckParts call.
type CheckPartsResp struct {
	// List of parts that are missing or otherwise invalid.
	MissingParts []int
}
```